ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -6,11 +6,14 @@ import {
   handleSettingsRefs,
 } from '../../../helpers/utils/settings-search';
 import { MetaMetricsEventCategory } from '../../../../shared/constants/metametrics';
-import { Text } from '../../../components/component-library';
+import { Text, Box } from '../../../components/component-library';
 import {
   FontWeight,
   TextColor,
   TextVariant,
+  Display,
+  FlexDirection,
+  JustifyContent,
```

Comment:
These additions should be in code fencing. It's resulting in the build error
![Screenshot 2023-08-03 at 2 16 05 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/be1e294d-c18b-4176-8873-00af16ec5025)